platform/viewer/src/components/ViewportGrid.tsx

```diff
@@ -263,11 +263,8 @@ function ViewerViewportGrid(props) {
     for (let i = 0; i < numViewportPanes; i++) {
       const viewportIndex = i;
       const isActive = activeViewportIndex === viewportIndex;
-      const paneMetadata = viewports[i] || {};
-      const viewportId = paneMetadata.viewportId || `viewport-${i}`;
-      if (!paneMetadata.viewportId) {
-        paneMetadata.viewportId = viewportId;
-      }
+      const paneMetadata = viewports[i] || { viewportId: `viewport-${i}` };
```

Comment:
I struggle as to when the viewport grid would have a `null` or `undefined` value for a viewport. Please fill me in. Thanks.